docker/Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN make && \
 VOLUME [ "/sys/fs/cgroup" ]
 
 # install keylime and have systemd run it at startup
-# have systemd start tpm_server, tpm2-abrmd and keyline-agent
+# have systemd start tpm_server, tpm2-abrmd and keylime-agent
```

Comment:
This is fine for this commit, but in general you want all of the changes in a commit to be related. And if there are typos in documentation or comments like this, it's better to have them as a separate commit so that they are super easy to review, approve and merge.

Just as an example, imagine in the future looking at this file (in `git blame` or something else) and seeing that it was changed in a commit titled "OKD setup + Package installer script". It wouldn't make much sense.

Reply:
Yes, I agree with you on this. I just came across these typos and felt it's too small of a change to make another PR for. I'll keep this in mind, thanks.